### ISSUE TITLE
Panic at 'Can only call resolve on linked references...`

### ISSUE THREAD

**Issue by davidarmstronglewis** [sharkdp/bat#190]
File name: `test.md`, file contents:

`` ```javascript ``
```var test = "boom";```
`` ``` ``

Here's a backtrace.

```
thread 'main' panicked at 'Can only call resolve on linked references: ByScope { scope: <source.javascript>, sub_context: None }', /Users/davidlewis/.cargo/registry/src/github.com-1ecc6299db9ec823/syntect-2.1.0/src/parsing/syntax_definition.rs:205:18
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic_fmt
   6: syntect::parsing::syntax_definition::ContextReference::resolve
   7: syntect::parsing::parser::ParseState::push_meta_ops
   8: syntect::parsing::parser::ParseState::parse_line
   9: syntect::easy::HighlightLines::highlight
 ```

I guess `syntect` is what's panicking, not `bat`. I'm on macOS 10.13.6 if you're unable to reproduce on your machine.

Thanks for the tool, by the way - I've been using it for a while and this is my first issue so far :)

**Comment by sharkdp** [comment]
Thank you for the feedback!


This is a duplicate of #183 which has recently been closed. This should be fixed in the next release (or on `master`, if you want to build it yourself)

**Comment by sharkdp** [comment]
Fixed in [bat 0.5](https://github.com/sharkdp/bat/releases/tag/v0.5.0).
